results/cmd/api/main_test.go

```diff
@@ -87,9 +96,7 @@ func TestGetTaskRun(t *testing.T) {
 	}
 
 	// Connect to fake server and create a new taskrun
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
-	defer cancel()
-	r, err := srv.CreateTaskRunResult(ctx, &pb.CreateTaskRunRequest{
+	r, err := srv.CreateTaskRunResult(nil, &pb.CreateTaskRunRequest{
```

Comment:
why did we switch from context.Background() to nil?

Reply:
Go tests have a timeout flag that can be configured, so we likely don't need to do this unless we have a particular reason to have a shorter per-request timeout.

Reply:
I decide to use context.Background only instead of nil

Reply:
done